databroker/src/grpc/kuksa_val_v2/val.rs

```diff
@@ -678,8 +680,30 @@ impl proto::val_server::Val for broker::DataBroker {
                                                     }
                                                 }
                                             },
-                                            Some(BatchActuateStreamResponse(_batch_actuate_stream_response)) => {
-                                                // TODO discuss and implement
+                                            Some(BatchActuateStreamResponse(batch_actuate_stream_response)) => {
+
+                                                if let Some(error) = batch_actuate_stream_response.error {
+                                                    match error.code {
+                                                        0 => {},
+                                                        _ => {
+                                                            let mut msg : String = "Batch actuate stream response error".to_string();
+                                                            if let Some(signal_id) = batch_actuate_stream_response.signal_id {
+                                                                match signal_id.signal {
+                                                                    Some(proto::signal_id::Signal::Path(path)) => {
+                                                                        msg = msg + ", path: " + &path;
+                                                                    }
+                                                                    Some(proto::signal_id::Signal::Id(id)) => {
+                                                                        msg = msg + ", id: " + &id.to_string();
+                                                                    }
+                                                                    None => {}
+                                                                }
+                                                            }
+                                                            msg = msg + ", error code: " + &error.code.to_string() + ", error message: " + &error.message;
+                                                            debug!(msg)
```

Comment:
Yes, I looked mostly at the Provider rpc. 

But I assume we need quite a lot refactoring if we are to return results to the "consumer", some aspects I think of:

* If `Actuate` / `BatchActuate` are to return actual results, the Databroker will need to wait until the providers has replied (for `BatchActuate` there could be multiple, right?)
* That means the providers MUST send a response also on OK. Possibly combined with a timeout in databroker so that Actuation is considered unsuccessful if Databroker does not get a response within X seconds.
* Then we maybe also need some form of the request-id/statefulness, so we know which request the error belong to.

Reply:
What you are discussing here was postponed if I remember correctly as we were not sure if we wanted to put databroker to wait until it receives ACK of receipt and forwards it to the consumer.
So in my opinion, this PR is enough for now if databroker can output some debug actuate information.